django/db/models/fields/__init__.py

```diff
@@ -969,11 +969,11 @@ def formfield(self, **kwargs):
             include_blank = not (self.has_default() or 'initial' in kwargs)
             defaults = {'choices': self.get_choices(include_blank=include_blank)}
         else:
-            form_class = forms.NullBooleanField if self.null else forms.BooleanField
+            form_class = forms.BooleanField
```

Comment:
Is this is the right commit? I don't think the deprecation of the model field should require any changes. I would have deprecated the model field first, then tackled the question of whether or not to deprecate the form field separately.

Reply:
If you're not sold on deprecating the form field, then I see why this would feel wrong. I've been implementing everything from the perspective they both are being deprecated.

Reply:
What I mean is that usually when deprecating a feature, we also remove usage of that feature. It feels like it was just an oversight that you put it in the third commit?

Reply:
No, I purposely put it in the third commit. I'm reworking it now to do the model field deprecation first with this change as a part of the form field deprecation.

The reasoning was this change felt like a part of deprecating the model field.